vllm/envs.py

```diff
@@ -218,6 +218,8 @@
     VLLM_USE_FBGEMM: bool = False
     VLLM_GC_DEBUG: str = ""
     VLLM_DISABLE_SHARED_EXPERTS_STREAM: bool = False
+    VLLM_USE_META_SHUFFLING_MOE: bool = False
+    VLLM_META_SHUFFLING_GEMM_BACKEND: Literal["cutlass", "triton"] = "cutlass"
```

Comment:
how about just infer VLLM_USE_META_SHUFFLING_MOE from VLLM_META_SHUFFLING_GEMM_BACKEND? We can have "none" for VLLM_META_SHUFFLING_GEMM_BACKEND.